### ISSUE TITLE
-B+N – new option to have annotations but omit frames

### ISSUE THREAD

**Issue by KristofKoch** [GenericMappingTools/gmt#4613]
**Description of the desired feature**

I want to make a plot with grid lines and annotations but no chart frame. Something similar to this example:

![gridlines_no_borders](https://user-images.githubusercontent.com/45796239/103230802-9d3cc800-4936-11eb-8f43-78ad559c01c7.jpg)

Currently there is no way to omit plot frames when annotations are desired. `-Ba` always switches on frames. @PaulWessel [suggested in his post in the forum](https://forum.generic-mapping-tools.org/t/gridlines-and-annotations-but-no-borders/1151/4) the implementation of `+N` to deal with this case. Please see the [discussion in the forum](https://forum.generic-mapping-tools.org/t/gridlines-and-annotations-but-no-borders/1151) for further background.

**Are you willing to help implement and maintain this feature?** Yes/No

Certainly if I can be of any help.

**Comment by KristofKoch** [comment]
Well … one possible workaround is to force plain borders and assign them a white pen. Didn't occur to me at the time. Would be nice to have a proper option for it none the less.

**Comment by PaulWessel** [comment]
Have not forgotten this one, @KristofKoch, but the proposal has consumed lots of time, and is due on Tuesday. Once that is submitted we will restart themes and get that merged and look at other issues.

**Comment by KristofKoch** [comment]
Hi @PaulWessel this was more meant as a heads up for other people stumbling over this – and as a note for you that this issue might be not so pressing after all as the workaround is rather simple. I keep my fingers crossed for the proposal!

**Comment by anbj** [comment]
This never left the drawing board, right? The best solution is still to draw the frame with a white pen?

**Comment by KristofKoch** [comment]
Yes, nothing has changed. You still need to draw the frame with a white pen.

**Comment by joa-quim** [comment]
Hi,
Nope, that was implemented on 6.6 (I think). Search for -B00 or something alike (away from computer this week).

Enviado a partir do Outlook para iOS<https://aka.ms/o0ukef>
________________________________
De: KristofKoch ***@***.***>
Enviado: Thursday, April 2, 2026 12:25:09 PM
Para: GenericMappingTools/gmt ***@***.***>
Cc: Subscribed ***@***.***>
Assunto: Re: [GenericMappingTools/gmt] -B+N – new option to have annotations but omit frames (#4613)

[https://avatars.githubusercontent.com/u/45796239?s=20&v=4]KristofKoch left a comment (GenericMappingTools/gmt#4613)<https://github.com/GenericMappingTools/gmt/issues/4613#issuecomment-4175895744>

Yes, nothing has changed. You still need to draw the frame with a white pen.

**Comment by anbj** [comment]
Very interesting. See https://docs.generic-mapping-tools.org/dev/gmt.html#axes-settings and https://docs.generic-mapping-tools.org/dev/reference/options.html#axes-settings:

_0 This will suppress all annotations and tick marks. Useful when you want to plot a map without any annotations, grid or ticks but still want to plot a frame using a line with set by [MAP_FRAME_PEN](https://docs.generic-mapping-tools.org/dev/gmt.conf.html#term-MAP_FRAME_PEN)._

_00 Plot a frame with annotations, ticks etc use a zero-width line to plot the frame (0-width lines are lines that remain very thin regardless of the zoom level)._

_000 Plot annotations, ticks and grid lines (as chosen) but do not plot the frame lines. Note, this effect is only visible with png and pdf formats and is not guaranteed to work when selecting only a subset of the axes (e.g., -B000ya30g30 is not working well)._

**Comment by joa-quim** [comment]
We can close this issue, right?

**Comment by anbj** [comment]
I have not tested this, so I do not know. Probably?

**Comment by anbj** [comment]
Did a small test today, and it works!

**Comment by Esteban82** [comment]
This can be achieved now for example with this

`gmt coast -W -RIT -png 1 -B000a2g2`

<img width="50%" alt="Image" src="https://github.com/user-attachments/assets/af51713c-6e06-4155-ba44-63062c449ddc" />
